internals/scripts/validate-config.js

```diff
@@ -1,12 +1,14 @@
 const { validator } = require('@exodus/schemasafe');
-const schema = require('../schemas/environment.conf.schema.json');
-const config = require('../../environment.conf.json');
+
```

Comment:
For the translation handling (#1039), I've rewritten this file so that not just the first validation error will be shown, but all the errors that were found. Helps with solving all mismatches in the schema at once. I'd propose changing this file's contents to:

```javascript
const { validator } = require('@exodus/schemasafe');

/* eslint-disable no-console, global-require */

const validationSections = {
  config: {
    schema: require('../schemas/app.schema.json'),
    json: require('../../app.json'),
  },
};

let hasValidationErrors = false;

Object.entries(validationSections).forEach(([section, { schema, json }]) => {
  const validate = validator(schema, { includeErrors: true, allErrors: true });

  if (!validate(json)) {
    console.error(`${section} is not valid`);
    console.table(validate.errors);

    hasValidationErrors = true;
  }
});

if (hasValidationErrors) {
  process.exit(1);
}
```

That way we can easily add other schemas at a later point, when needed.

Reply:
Including all errors is a good idea!
It nice that we could easily add more schema's with this script. However, at the moment we have only one schema to check and I am not a fan of adding logic and complexity that might be used in the future, while it is not necessary now.